cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineStopButton/PipelineStopPopover.js

```diff
@@ -164,16 +178,18 @@ export default class PipelineStopPopover extends Component {
                       key={i}
                       className={classnames({"current-run-row": run.runid === this.props.currentRunId})}
                     >
-                      <td>
+                      <td onClick={this.navigateToRun.bind(null, run.runid)}>
                         {
                           run.runid === this.props.currentRunId ?
                             <IconSVG name="icon-check" />
                           :
                             null
                         }
                       </td>
-                      <td>{runStartTime}</td>
-                      <td>
+                      <td onClick={this.navigateToRun.bind(null, run.runid)}>
+                        {runStartTime}
+                      </td>
+                      <td onClick={this.navigateToRun.bind(null, run.runid)}>
```

Comment:
Rather than attaching an onClick, can we turn this into an anchor tag an just compute the href link for it?